### PR TITLE
grant extra access to services, fixes #46

### LINKED ISSUE #46: Add new policies to service pod
`identity/oidc/role/*`
`identity/oidc/key/*`
`auth/$cluster/role/*`
`identity/oidc/token/*`

### PR DESCRIPTION
**Pull request by snripa**
fixes #46

### CODE REVIEW

File: app/common/vault_api.py [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import os
 
 import hvac
+from jinja2 import Environment, FileSystemLoader
```
Comment (artem-v) [inline app/common/vault_api.py:4]: jinja .. funny name

File: app/common/templates/service-policies.j2 [hunk 2]
```diff
@@ -0,0 +1,6 @@
+path "{{secrets_root}}/{{owner}}/{{repo}}/*" { capabilities = ["create", "read", "update", "delete", "list"]}
+
+path "identity/oidc/key/*" { capabilities = ["create", "read", "update", "delete", "list"]}
+path "identity/oidc/role/*" { capabilities = ["create", "read", "update", "delete", "list"]}
+path "auth/{{mount_point}}/role/*" { capabilities = ["create", "read", "update", "delete", "list"]}
+path "identity/oidc/token/*" { capabilities = ["create", "read", "update", "delete", "list"]}
```
Comment (artem-v) [inline app/common/templates/service-policies.j2:6]: `/*` -- dangerous and powerful thing at the same time. Let's not forget to change it later.

**Review by artem-v — APPROVED** [review]

File: app/helm/templates/service-policies.j2 [hunk 3]
```diff
@@ -1,29 +0,0 @@
...
-  labels:
-    app: {{ app }}
-  name: aws-reclaim
-parameters:
-  fsType: ext4
-  type: gp2
-  zone: eu-west-2a
-provisioner: kubernetes.io/aws-ebs
-reclaimPolicy: Retain
-volumeBindingMode: Immediate
```
Comment (artem-v) [inline app/helm/templates/service-policies.j2:29]: What was that? Why it was deleted?

**Review by artem-v — APPROVED** [review]

File: app/common/vault_api.py [hunk 4]
```diff
@@ -13,9 +13,11 @@
 class Vault:
     def __init__(self, logger,
                  owner=None,
-                 repo=None):
+                 repo=None,
+                 cluster_name=''):
         self.owner = owner
         self.repo = repo
+        self.mount_point = f'kubernetes-{cluster_name}'
```
Comment (artem-v) [inline app/common/vault_api.py:20]: So u know both cluster_name and mount point. Cool.

**Review by artem-v — APPROVED** [review]

File: app/helm/templates/service-policies.j2 (hunk 3 above)
Comment (snripa) [reply app/helm/templates/service-policies.j2:29]: accidental copy paste in prev. commit

File: app/common/vault_api.py (hunk 4 above)
Comment (snripa) [reply app/common/vault_api.py:20]: sure

**Pull request merged**